api.planx.uk/.env.test.example

```diff
@@ -45,3 +45,7 @@ ORDNANCE_SURVEY_API_KEY=👻
 
 IDOX_NEXUS_TOKEN_URL=👻
 IDOX_NEXUS_SUBMISSION_URL=👻
+
+# Analytics
+METABASE_API_KEY=👻
```

Comment:
We also need to add this to `.env.example`.

`.env.example` is a public list of the required secrets needed to run PlanX, and serves as documentation.

`.env.test.example` is a per-project subset of the full env var list, which only includes variables required to run unit tests.

Reply:
I think they're [in there already](https://github.com/theopensystemslab/planx-new/blob/06e01756676637d0fbb433a15c859d17454b5e5a/.env.example#L54C1-L57C51) from earlier PR #3815

Reply:
Ah perfect - thanks!